solr-ingest/src/test/java/edu/unc/lib/dl/data/ingest/solr/filter/SetAccessStatusFilterTest.java

```diff
@@ -145,6 +145,7 @@ public void testHasStaffOnlyAccess() throws Exception {
 
```

Comment:
Could you add a check in `testHasInheritedPublicAccess` to verify
`assertFalse(listCaptor.getValue().contains(FacetConstants.PATRON_SETTINGS));`
?

It may also make sense to add a test where both the parent and the child have patron settings, to ensure that the child still gets PATRON_SETTINGS.